pyrdp/mitm/X224MITM.py

```diff
@@ -83,6 +83,9 @@ def onConnectionRequest(self, pdu: X224ConnectionRequestPDU):
             # Tell the server we only support the allowed authentication methods.
             chosenProtocols &= self.state.config.authMethods
 
+        if self.state.ntlmCapture:
+            chosenProtocols = NegotiationProtocols.SSL | NegotiationProtocols.CRED_SSP
```

Comment:
I think we need to describe why we added that here in a comment. Here's a suggestion:

```suggestion
            # If we want to capture the NTLM hash, we need to put back CredSSP in here.
            # If we don't do that we will not get to the state where we can clone the certificate if needed.
            chosenProtocols = NegotiationProtocols.SSL | NegotiationProtocols.CRED_SSP
```